image/cli/mascli/functions/gitops_mas_config

```diff
@@ -15,6 +15,7 @@ GitOps Configuration:
   -d, --dir ${COLOR_YELLOW}GITOPS_WORKING_DIR${TEXT_RESET}                     Working directory for GitOps repository
   -a, --account-id ${COLOR_YELLOW}ACCOUNT_ID${TEXT_RESET}                      Account name that the cluster belongs to
   -c, --cluster-id ${COLOR_YELLOW}CLUSTER_ID${TEXT_RESET}                      Cluster ID
+  -s ,--sls-service ${COLOR_YELLOW}STANDALONE_SLS_SERVICE${TEXT_RESET}         SLS service parameter file Path.
```

Comment:
description doesn't make much sense. What file path? Can you give examples in the description.